runtime/cudaq/platform/default/rest/helpers/ionq/ionq.config

```diff
@@ -0,0 +1,22 @@
+# ============================================================================ #
+# Copyright (c) 2022 - 2023 NVIDIA Corporation & Affiliates.                   #
+# All rights reserved.                                                         #
+#                                                                              #
+# This source code and the accompanying materials are made available under     #
+# the terms of the Apache License 2.0 which accompanies this distribution.     #
+# ============================================================================ #
+
+# Tell DefaultQuantumPlatform what QPU subtype to use
+PLATFORM_QPU=remote_rest
+
+# Tell NVQ++ to generate glue code to set the target backend name
+GEN_TARGET_BACKEND=true
+
+# Add the rest-qpu library to the link list
+LINKLIBS="${LINKLIBS} -lcudaq-rest-qpu"
+
+# Define the lowering pipeline
+PLATFORM_LOWERING_CONFIG="expand-measurements,canonicalize,cc-loop-unroll,canonicalize,func.func(lower-to-cfg),canonicalize"
```

Comment:
I think you'll want to add `,ionq-gate-set-mapping` here

Reply:
got it :)